docs/getting-started.md

```diff
@@ -72,15 +72,22 @@ few things. Most important is the `opensafely` tool; to install this, you must
    these instructions](https://docs.docker.com/docker-for-windows/install/).
    Otherwise, follow the instructions for [installing on Windows
    Home](https://docs.docker.com/docker-for-windows/install-windows-home/).
+   Unfortunately, we've had reports that installing in Windows Home can be very
+   challenging. Please let us know if you can help us improve the documentation.
+   * Note that there are two kinds of "backend" in Docker for Windows: one called
+     *Hyper-V* and another called *WSL 2*. Where supported, the default is *WSL
+     2*; you can accept all the related defaults during the install process.
 2. Starting Docker can take a while &mdash; up to 5 minutes. While it's doing
    so, an animation runs in the notification area:<br>
    ![image](images/docker-windows-starting.png)
-3. When Docker has finished starting up, share your hard drive with Docker:
-   click system tray docker icon; select "settings"; select "shared drives".
+3. If you have ended up with the *Hyper-V* backend, then when Docker has
+   finished starting up, you will need share your hard drive with Docker: click
+   system tray docker icon; select "settings"; select "shared drives".  This
+   setting does not exist in the *WSL 2* backend.
 4. Test Docker and opensafely work together. Open an Anaconda Prompt, and run
-   `opensafely pull`.  The first time you run it, this may take a long time,
-   depending on your network connection. It is downloading a reproducible
-   environment identical to that installed in the OpenSAFELY secure
+   `opensafely pull --force`.  The first time you run it, this can take a very
```

Comment:
I would remove this step entirely. `opensafely run` will download any images it needs.  There's no need to force the user to download every image (~10Gb) when they are only going to use `cohortextractor` image (~1GB) in this getting started guide.

If you really want to pre-load it, do `opensafely pull cohortextractor`